homeworks/sasha.demidenko_SashaDemi/GamesProject/scripts/games.js

```diff
@@ -188,7 +61,7 @@ function filterOldGames(allGames) {
     return oldGames;
 }
 
-function displayCards(container, gamesList, isNewChecked, isOldChecked) {
+function displayCards(cont, gamesList, isNewChecked, isOldChecked) {
```

Comment:
... and here you don't use param `cont` in the function. On lines 85-86, you get it for the outer scope variable `container`
Please keep only one container variable.
Actually, it needs only for this function, so you can move it inside
```suggestion
function displayCards(gamesList, isNewChecked, isOldChecked) {
const container = document.querySelector('...
```